lib/palletjack/pallet.rb

```diff
@@ -48,11 +48,10 @@ def load(identity)
           edge(pallet, pallet:{references:{file => link_id.full_name}})
         when filestat.directory?
           child = jack.pallet(kind, File.join(name, file))
-          child.edge(self, pallet:{references:{_parent: name}})
+          child.edge(self, pallet:{references:{_parent: name}}})
```

Comment:
Fixing this syntax error in a later commit is not an excuse for pushing it in the first place.